infrastructure/lib/dynamodb-stack.ts

```diff
@@ -0,0 +1,26 @@
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { AttributeType, BillingMode, Table } from '@aws-cdk/aws-dynamodb';
+
+export interface DynamoDBStackProps extends StackProps {
+}
+
+export class DynamoDBStack extends Stack {
+  readonly table: Table;
+
+  constructor(scope: App, id: string, props: DynamoDBStackProps) {
+    super(scope, id, props);
+
+    this.table = new Table(this, 'cribbageDynamoTableID', {
+      tableName: 'cribbage',
+      partitionKey: {
+        name: 'DDBid',
```

Comment:
Before we go an deploy this out and lock ourselves into this partition key name... Is this a good name? Maybe should have been a comment on the last PR, but here we are :shrug:

Reply:
CDK makes it easy enough to re-deploy (and re-create our DB:#). Let's address with https://github.com/joshprzybyszewski/cribbage/issues/112

Reply:
Right, I was mostly just worried about losing a bunch of data if we decide to do that much later. But doing in a followup sounds good